### PR TITLE
Improve install-standalone and install-profile scripts

### PR DESCRIPTION
**Pull request by Clumsy-Coder**
- install-standalone
  - [x] check if config file exist
  - [x] print a banner line before running config
  - [x] load shell ENV from a file (currently not being used)
    - The code is implemented, it not enabled
  - [x] Add documentation on how the script works
- install-profile
  - [x] simplify code
    - remove redundant code of `install-standalone`
    - use a single command to run profile configs
  - [x] check if profile exists
  - [x] have the ability to run multiple profiles in a single command
  - [x] print line banner before running profile
  - [x] ignore certain lines in a profile
    - Allows the ability to omit a config from running in a profile
  - [x] Add documentation on how the script works
  - [x] Remove unused code
    - [x] Remove unused ENVs
- miscellaneous
  - [x] add example config
  - [x] add example profile
- Update README
  - [x] running a profile
    - [x] general info for running a profile
    - [x] running single profile
    - [x] running multiple profiles
    - [x] ignore certain lines in a profile
  - [x] running a config
    - [x] for general info for running a config
    - [x] for running single config
    - [x] for running multiple configs
    - [x] for running config in sudo
  - [x] using ENVs
    - [x] general info on using ENVs
    - [x] why use ENVs
    - [x] how to enable
  - [x] update file tree

### CODE REVIEW

**Comment by Clumsy-Coder** [comment]
Pull request connected to #7

**Comment by kiriwalawren** [comment]
I'm sorry it has taken me a while to get to this. I am in the process of moving. I will try to review this PR sometime this week.

File: README.md [hunk 1]
```diff
@@ -8,18 +9,30 @@ The structure of this repository is heavily inspired by [vsund](https://github.c
 ## Table of Contents
 <!-- TOC GFM -->
 
-- [Dependencies](#dependencies)
-- [Installation](#installation)
+- [Dotbot Template](#dotbot-template)
+  - [Table of Contents](#table-of-contents)
```
Comment (kiriwalawren) [inline README.md]: We don't need these.
```suggestion
```

**Review by kiriwalawren — CHANGES_REQUESTED** [review]
Good job. I appreciate the documentation. Only thing I can see is that there are 2 unnecessary additions to the table of contents.

How have you tested these changes?

File: README.md (hunk 1 above)
Comment (kiriwalawren) [reply README.md]: The rest will need to be de-dented after the removal of the above.

Comment (Clumsy-Coder) [reply README.md]: Made changes as per requested.

Check [fb88d4f](https://github.com/ecarlson94/dotbot-template/pull/9/commits/fb88d4f5033a76023ff3644ad71c2b4bf7a6eafc)

**Comment by Clumsy-Coder** [comment]
> How have you tested these changes?

# install-standlone
## running multiple configs in one command

Running `./install-standalone example-config-1 example-config-2-sudo` as proof it works

## Exiting if attempting to run non-existent config file

Run `./install-standalone non-existent-config`

Works with sudo as well. Run `./install-standalone non-existent-config-sudo`

It will report that this config doesn't exist and exit
(it will make sure all config files in the command exist before running them)

# install-profile

## running multiple profiles in one command

Running `./install-profile exampleOS1 exampleOS2` as proof it works

## Exiting if attempting to run non-existent profile

Run `./install-profile non-existent-profile`

It will report that this profile doesn't exist and exit
(it will make sure all profiles in the command exist before running them)

**Review by kiriwalawren — APPROVED** [review]

**Pull request merged**